README.md

```diff
@@ -53,14 +53,13 @@ Add the following to your `tmux.conf` file to configure the tmux side of
 this customization.
 
 ``` tmux
-# Smart pane switching with awareness of vim splits
+# Smart pane switching with awareness of Vim splits.
 # See: https://github.com/christoomey/vim-tmux-navigator
-is_vim='echo "#{pane_current_command}" | grep -iqE "(^|\/)g?(view|n?vim?x?)(diff)?$"'
-bind -n C-h if-shell "$is_vim" "send-keys C-h" "select-pane -L"
-bind -n C-j if-shell "$is_vim" "send-keys C-j" "select-pane -D"
-bind -n C-k if-shell "$is_vim" "send-keys C-k" "select-pane -U"
-bind -n C-l if-shell "$is_vim" "send-keys C-l" "select-pane -R"
-bind -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
+bind -n C-h if-shell "~/.dotfiles/vim/neobundles/tmux-navigator/is_vim.sh #{pane_pid}" "send-keys C-h" "select-pane -L"
```

Comment:
This could/should use `/path/to/is_vim.sh` probably?
It's shorter and needs to be changed in most cases anyway.